packages/sdk-components-react/src/vimeo-preview-image.ws.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  type WsComponentMeta,
+  type WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+import { propsMeta as imagePropsMeta, meta as imageMeta } from "./image.ws";
+import { props } from "./__generated__/vimeo-preview-image.props";
+
+export const meta: WsComponentMeta = {
+  ...imageMeta,
+  category: undefined,
```

Comment:
Can it be omitted?

Reply:
or alternatively add "none" for no category, so that's it's nicer to use?

Reply:
added category "hidden"